backend/src/api/routes/moving_averages.py

```
@@ -0,0 +1,345 @@
+            result = await asyncio.wait_for(
+                service.get_metric_with_ma(
+                    workspace_id=workspace_id,
+                    metric=metric,
+                    ma_type=ma_type,
+                    window=window,
+                    timeframe=timeframe,
+                    weights=parsed_weights
+                ),
+                timeout=CALCULATION_TIMEOUT_SECONDS
+            )
+            return result
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)